### DISCUSSION TITLE
Bricked Laptop

### DISCUSSION THREAD

**Discussion by FlareSwap** [DavidS95/Smokeless_UMAF#40] [General]
Any tips on recovery a bricked laptop after using this tool ? 
My laptop is Asus Rog Strix G733QS. All lights are on but the screen is blank.
Thanks.

**Comment by pyrokzg** [comment]
a year late I know, I think you may have caused a short in your laptop.

**Reply by VoltexFPS** [reply]
I've also disabled a couple of known bad settings. Though I researched every change and made sure they were all safe, I somehow still managed to brick my laptop. The symptoms are the same as FlareSwap's above. Please note that I haven't done any overclocking or memory tuning, and I also declined all risk warnings.

**Comment by Blaubluter** [comment]
Why do you even need a BIOS hack with such a good model, what did you want to do?.. lol well too late


Yes

 

On/Off button pressed for 50 seconds , Self  Bios  Repair.

**Comment by zamroni111** [comment]
My ThinkPad t14s g4  is bricked.

I was just exploring the lpddr menu.
No saving changes were done.
I esc to main menu, reboot using ctrl alt del, then screen blank.

**Reply by Blaubluter** [reply]
no Intel  ?  AMD BIOS MOD ! 

 On AMI Bios, in addition to AMD PBS and CBS 
AMD Overclocking menu -  
AMD PBS/CBS, Aod Setup)-


Lenovo ThinkPad T14s G4 14"2.8K"""" >>>>  i7-1355U 32GB Intel ?

--

**Reply by zamroni111** [reply]
> no Intel ? AMD BIOS MOD !
> 
> On AMI Bios, in addition to AMD PBS and CBS AMD Overclocking menu - AMD PBS/CBS, Aod Setup)-
> 
> Lenovo ThinkPad T14s G4 14"2.8K"""" >>>> i7-1355U 32GB Intel ?
> 
> --

It's amd 7840u

**Reply by Blaubluter** [reply]
Can you take out the battery? Battery and bios Battery

**Reply by zamroni111** [reply]
> Can you take out the battery? Battery and bios Battery

I have disconnected battery and the cmos battery, but still blank screen no post

**Reply by Blaubluter** [reply]
Are you waiting long enough?  It took a very long time until a picture was there 

Is this you Bios Style ?

https://support.lenovo.com/de/de/solutions/ht105352-how-to-turn-off-unexpected-auto-power-on-thinkpad

**Reply by zamroni111** [reply]
> Are you waiting long enough? It took a very long time until a picture was there
> 
> Is this you Bios Style ?
> 
> https://support.lenovo.com/de/de/solutions/ht105352-how-to-turn-off-unexpected-auto-power-on-thinkpad

I left it for 5 minutes and still blank.

Yes, usual Bios menu display was like that

**Reply by Blaubluter** [reply]
It could be an absolute coincidence. That it broke right now so I would say RMA

**Reply by manualemartinez** [reply]
> I left it for 5 minutes and still blank.
> 
> Yes, usual Bios menu display was like that

Did you manage to fix this?
